src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectManagementService.cs

```diff
@@ -66,11 +65,10 @@ internal async Task HandleRenameRequest(RenameRequestParams requestParams, Reque
                     requestParams.ConnectionUri,
                     out connInfo))
             {
-                using (SqlConnection sqlConn = ConnectionService.OpenSqlConnection(connInfo, ObjectManagementServiceApplicationName))
+                ServerConnection serverConnection = ConnectionService.OpenServerConnection(connInfo, ObjectManagementServiceApplicationName);
```

Comment:
Does this imply there's a bug with `OpenSqlConnection`? Shouldn't that be keeping all the same info regardless?

Reply:
no, OpenSqlConnection is fine, the problem is with how we construct the ServerConnection using SqlConnection in this service. we didn't include the token information like the `OpenServerConnection()`

![image](https://user-images.githubusercontent.com/13777222/225447072-a4182583-12de-4cc3-a14f-fa253b2f78e2.png)